workspaces/adventure-pack/goodies/java/src/digits_int/Test.java

```diff
@@ -0,0 +1,72 @@
+package digits_int;
+
+import static org.junit.jupiter.api.Assertions.assertIterableEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import iterable_int_stream.IterableIntStream;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.Test;
+
+class DigitsIntTest {
+
+  @Test
+  public void withIntegerGreaterThanZero() {
+    try {
+      var digits = AP.digits(4984);
+      assertIterableEquals(
+        IterableIntStream.from(IntStream.of(4, 8, 9, 4)),
```

Comment:
does `List.of(4, 8, 9, 4)` work here?

Reply:
It won't work if you mean `IterableIntStream.from(List.of(4, 8, 9, 4))` as we do not support List argument for `IterableIntStream.from()` method. But, we can assert equality for arrays like,
```
var digits = digits(4984).toArray();
var expected = {4,8,9,4};
assertArrayEquals(expected, digits);
```
does it makes sense?

Reply:
I meant something like:

```java
assertIterableEquals(
  List.of(4, 8, 9, 4),
  digits
);
```

From the name `assertIterableEquals` I am assuming it accepts any `Iterable`, and a `List` is an `Iterable`.

Reply:
Got it. I'll check.

Reply:
Addressed.